src/components/buttons/analytic-router-link.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <router-link :to="to" @click="handleClick">
+    <slot> </slot>
```

Comment:
```suggestion
    <slot />
```

C'est la même chose en plus concis